SharedPackages/VPN/Sources/VPN/Routing/VPNRoutingRange.swift

```diff
@@ -56,7 +56,12 @@ public enum VPNRoutingRange {
         "12.0.0.0/6",       /* 12.0.0.0 - 15.255.255.255 */
         "16.0.0.0/4",       /* 16.0.0.0 - 31.255.255.255 */
         "32.0.0.0/3",       /* 32.0.0.0 - 63.255.255.255 */
-        "64.0.0.0/2",       /* 64.0.0.0 - 127.255.255.255 */
+        "64.0.0.0/3",       /* 64.0.0.0 - 95.255.255.255 */
+        "96.0.0.0/4",       /* 96.0.0.0 - 111.255.255.255 */
+        "112.0.0.0/5",      /* 112.0.0.0 - 119.255.255.255 */
+        "120.0.0.0/6",      /* 120.0.0.0 - 123.255.255.255 */
+        "124.0.0.0/7",      /* 124.0.0.0 - 125.255.255.255 */
+        "126.0.0.0/8",      /* 126.0.0.0 - 126.255.255.255 */
         /* 127.0.0.0            127.0.0.0/8 - 127.255.255.255   Loopback */
```

Comment:
Bug 1: loopback was included.  This wasn't an issue though because loopback is explicitly excluded and takes priority.

A test was added to validate this.